script.js

```diff
@@ -0,0 +1,40 @@
+const conteiner = document.querySelector(".conteiner");
+const leftArrow = document.querySelector(".left");
+const rightArrow = document.querySelector(".right");
+
+const quantityElemnts = parseInt(prompt("How much elements would you like to see?", "cancell for default"));
+if (typeof quantityElemnts === 'number' && !isNaN(quantityElemnts)) {
+  alert("IN!")
+  conteiner.style.width = quantityElemnts * 130 + "px";
```

Comment:
if you use --conteiner-width
also you can use --quantityElemnts and set css property from js instead of width